static/css/external-site.less

```diff
@@ -0,0 +1,46 @@
+/* ==========================================================================
+   cfgov-refresh
+   Third-Party links pages at /external-site/.
+   ========================================================================== */
+
+.external-site_container {
+    .content_bar {
+      border-top: 1px solid @gray-50;
+      height: 0;
+    }
+
+    .padded-container {
+       background-color: @white;
```

Comment:
Are the padded-containers not white by default? (I forget where these are defined)